spec/gemstash/cli/authorize_spec.rb

```diff
@@ -116,6 +136,18 @@
       Gemstash::CLI::Authorize.new(cli).run
       expect(Gemstash::Authorization["auth-key"]).to be_nil
     end
+
+    it "combined with --list results in an error" do
+      cli_options[:list] = true
+      expect { Gemstash::CLI::Authorize.new(cli).run }.to raise_error(Gemstash::CLI::Error)
+      expect(Gemstash::Authorization["auth-key"]).to be # Auth was not actually removed
```

Comment:
TIL about this very bare form of Matcher.

Reply:
Same. I stole it from another test! One of the beautiful things about open source :heart: .